tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -244,3 +245,16 @@ func (mcs *MultiClientSnapshotter) createOrGetSnapshotter(ctx context.Context) (
 
 	return cs, nil
 }
+
+// GetCacheDirInfo runs cache info command to get cache dir path for
+// the repository.
+func (mcs *MultiClientSnapshotter) GetCacheDirInfo() (stdout, stderr string, err error) {
+	stdout, stderr, err = mcs.server.Run("cache", "info", "--path")
```

Comment:
if `kopia cache info` shows all the paths, why get the paths one by one?

Reply:
I tried parsing the output of `kopia cache info`, it's strung together on the go and the parsing code became complicated and really targeted towards the format. If the printing format of `kopia cache info` were to change even a little bit, the logic would fail. The command does not support `--json` at the moment.
The command provides predictable output with `--path` flag. It is easier to parse, the parsing code is easy to read and maintain. Introducing a new type of cache won't break the existing code.